packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -107,7 +107,10 @@
     }
 
     div.paragraph {
-        .legacy-typography;
+        font-family: '"Noto Serif", serif';
```

Comment:
Will this mean we need to ship new fonts with our mobile app?

Reply:
We should have these through wonderblocks. Do we need to add these to mobile? I would be surprised if any of perseus looked right without mostly having these baked in?

Reply:
🤔  It looks like we have Noto for some non-en languages, but I don't see a base Noto font included anywhere in our codebase:

https://github.com/Khan/mobile/blob/develop/react-native/wonderblocks-rn/core/typography.js#L33

and 

```sh
$ find . -name "*Noto*"
./ios/Resources/NotoSansDevanagari-Light.ttf
./ios/Resources/NotoSansGeorgian-Light.ttf
./ios/Resources/NotoSansBengali-Bold.ttf
./ios/Resources/NotoSansArmenian-Bold.ttf
./ios/Resources/NotoSansArmenian-Thin.ttf
./ios/Resources/NotoSansArmenian-Regular.ttf
./ios/Resources/NotoSansDevanagari-Bold.ttf
./ios/Resources/NotoSansBengali-Black.ttf
./ios/Resources/NotoSansBengali-Regular.ttf
./ios/Resources/NotoSansArmenian-Black.ttf
./ios/Resources/NotoSansBengali-Light.ttf
./ios/Resources/NotoSansDevanagari-Regular.ttf
./ios/Resources/NotoSansGeorgian-Bold.ttf
./ios/Resources/NotoSansGeorgian-Regular.ttf
./ios/Resources/NotoSansGeorgian-Black.ttf
./ios/Resources/NotoSansDevanagari-Black.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Light.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Light.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Thin.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Black.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Black.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Light.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Black.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Black.ttf

$ 
```

Reply:
Here's a ticket to make sure this work gets done when we deploy on mobile: https://khanacademy.atlassian.net/browse/MOB-4782